.github/workflows/crucible-ci.yaml

```diff
@@ -0,0 +1,37 @@
+name: crucible-ci
+
+on:
+  # run on pull request events for the master branch only
+  pull_request:
+    branches: [ master ]
+
+  # allow for manual invocation from the actions tab
+  workflow_dispatch:
+
+jobs:
+  crucible-ci:
+    runs-on: [self-hosted, cpu-partitioning]
+
+    steps:
+    - name: Checkout bench-oslat
+      uses: actions/checkout@v2
+      with:
```

Comment:
repository and ref are optional?

Reply:
I believe it defaults to the current project and it's primary branch